File: php-breeds/app/models/Dog.php
<?php

Class Dog {
	private $id;
	private $name;
	private $name_es;
	private $breed_id;
	private $family;
	private $description;
	private $description_es;
	private $year_of_birth;
	private $sex;
	private $address;
	private $vaccines;
	private $path_image;
	private $date;
	private $creator_id;
	
	/**
	 * CONSTRUCTOR
	 */
	function __construct($id){
		$this->id=$id;
	}
	

	/**
	 * SETTERS
	 */
	function set_name($name){
		$this->name=$name;
	}
    function set_name_es($name_es){
		$this->name_es=$name_es;
	}
    function set_breed_id($breed_id){
		$this->breed_id=$breed_id;
	}
    function set_family($family){
		$this->family=$family;
	}
	function set_description($description){
		$this->description=$description;
	}
	function set_description_es($description_es){
		$this->description_es=$description_es;
	}
	function set_year_of_birth($year_of_birth){
		$this->year_of_birth=$year_of_birth;
	}
	function set_sex($sex){
		$this->sex=$sex;
	}
	function set_address($address){
		$this->address=$address;
	}
	function set_vaccines($vaccines){
		$this->vaccines=$vaccines;
	}
	function set_path_image($path_image){
		$this->path_image=$path_image;
	}
    function set_date($date){
		$this->date=$date;
	}
    function set_creator_id($creator_id){
		$this->creator_id=$creator_id;
	}
    /**
	 * GETTERS
	 */
	function get_id(){
		return $this->id;
	}
	function get_name(){
		return $this->name;
	}
    function get_name_es(){
		return $this->name_es;
	}
    function get_breed_id(){
		return $this->breed_id;
	}
    function get_family(){
		return $this->family;
	}
	function get_description(){
		return $this->description;
	}
	function get_description_es(){
		return $this->description_es;
	}
	function get_year_of_birth(){
		return $this->year_of_birth;
	}
	function get_sex(){
		return $this->sex;
	}
	function get_address(){
		return $this->address;
	}
	function get_vaccines(){
		return $this->vaccines;
	}
	function get_path_image(){
		return $this->path_image;
	}
    function get_date(){
		return $this->date;
	}
    function get_creator_id(){
		return $this->creator_id;
	}

    	
	/**
	 * TODTRING
	 */
	function to_String(){
		return"<br><img src=".PATHSERVERSININDEX.$this->path_image." width=40px>Dog id: ".$this->id.", name: ".$this->name.", breed id: ".$this->breed_id.", path: ".$this->path_image.", creator: ".$this->creator_id;	
	}
}

?>

File: java-ConsumoAPI/src/main/java/com/mycompany/consumoapi/models/Dog.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.consumoapi.models;

/**
 *
 * @author kikem
 */
public class Dog {
    private int id;
    private String name;
    private String name_es;
    private int breed_id;
    private String family;
    private String description;
    private String year_of_birth;
    private String sex;
    private String address;
    private String vaccines;
    private String path_image;
    private String date;
    private int creator_id;

    public Dog(int id, String name, String name_es, int breed_id, String family, String description, String year_of_birth, String sex, String address, String vaccines, String path_image, String date, int creator_id) {
        this.id = id;
        this.name = name;
        this.name_es = name_es;
        this.breed_id = breed_id;
        this.family = family;
        this.description = description;
        this.year_of_birth = year_of_birth;
        this.sex = sex;
        this.address = address;
        this.vaccines = vaccines;
        this.path_image = path_image;
        this.date = date;
        this.creator_id = creator_id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName_es() {
        return name_es;
    }

    public void setName_es(String name_es) {
        this.name_es = name_es;
    }

    public int getBreed_id() {
        return breed_id;
    }

    public void setBreed_id(int breed_id) {
        this.breed_id = breed_id;
    }

    public String getFamily() {
        return family;
    }

    public void setFamily(String family) {
        this.family = family;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getYear_of_birth() {
        return year_of_birth;
    }

    public void setYear_of_birth(String year_of_birth) {
        this.year_of_birth = year_of_birth;
    }

    public String getSex() {
        return sex;
    }

    public void setSex(String sex) {
        this.sex = sex;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getVaccines() {
        return vaccines;
    }

    public void setVaccines(String vaccines) {
        this.vaccines = vaccines;
    }

    public String getPath_image() {
        return path_image;
    }

    public void setPath_image(String path_image) {
        this.path_image = path_image;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public int getCreator_id() {
        return creator_id;
    }

    public void setCreator_id(int creator_id) {
        this.creator_id = creator_id;
    }

    @Override
    public String toString() {
        return "Dog{" + "id=" + id + ", name=" + name + ", name_es=" + name_es + ", breed_id=" + breed_id + ", family=" + family + ", description=" + description + ", year_of_birth=" + year_of_birth + ", sex=" + sex + ", address=" + address + ", vaccines=" + vaccines + ", path_image=" + path_image + ", date=" + date + ", creator_id=" + creator_id + '}';
    }


    
    
    
}
